help-center/faq/memgraph-lab-faq.md

```diff
@@ -0,0 +1,57 @@
+---
+id: memgraph-lab-faq
+title: Memgraph Lab - Frequently asked questions
+sidebar_label: Memgraph Lab
+slug: /faq/memgraph-lab
+---
+
+### What is Memgraph Lab?
+
+Memgraph Lab is a visual interface for exploring and interacting with data
+stored in Memgraph.
+
+If you prefer to query Memgraph using a command-line tool, use
+[mgconsole](https://memgraph.com/docs/memgraph/connect-to-memgraph/mgconsole). 
+
+### Can I only use Memgraph Lab?
+
+No. Memgraph Lab needs a running Memgraph instance in the background to be able to
+connect to it. After you have [installed Memgraph
+platform](/memgraph/installation) and started it, you can head over to Memgraph
+Lab and start querying your graph.
+
+### How to install Memgraph Lab on Windows?
+
+Memgraph Lab cannot be installed natively on Windows devices but can be installed

```

Comment:
@vpavicic Memgraph Lab can be installed on Windows natively (and on macOS and Linux). The one that can't be installed natively on Windows and macOS is Memgraph DB. The answer here should say that while you can install Memgraph Lab, you will need either Docker or WSL to install it Memgraph on Windows.

Reply:
This is some C/P nuclear accident :)) will fix